src/pages/NewSignup.vue

```diff
@@ -39,36 +39,53 @@
         :maxLengthOfEntry="formField.maxLengthOfEntry"
         :helpText="formField.helpText[getLocale]"
         @update="updateUserData"
+        class="mt-[25px]"
+      />
+
+      <!-- submit button -->
+      <button
+        class="mt-[30px] bg-primary disabled:bg-primary-hover hover:bg-primary-hover text-white mx-auto shadow-md w-full p-2 rounded"
+        :disabled="signUpDisabled"
+        @click="signUp"
+        v-html="signUpButtonLabel"
       />
     </div>
-    <!-- submit button -->
+    <div class="mt-[30px] flex w-48 mx-auto justify-between items-center">
+      <hr class="w-20 text-grey" />
+      <p class="text-grey font-roboto text-sm opacity-40">or</p>
+      <hr class="w-20 text-grey" />
+    </div>
+    <!-- signup button -->
     <button
-      class="bg-primary hover:bg-primary-hover text-white font-bold shadow-xl uppercase text-lg mx-auto p-4 rounded disabled:opacity-50 btn"
-      :disabled="signUpDisabled"
-      @click="signUp"
-    >
-      SIGN UP
-    </button>
+      @click="redirectToSignIn"
+      class="mt-[20px] mx-auto pt-2 text-primary text-base"
+      v-html="signInText"
+    />
   </div>
   <div
     v-if="formSubmitted"
-    class="w-5/6 lg:w-1/2 mx-auto flex flex-col bg-peach text-center mt-20 shadow-sm justify-evenly text-lg md:text-xl rounded-md p-6 space-y-6"
+    class="w-5/6 lg:w-1/2 mx-auto flex flex-col text-center mt-20 justify-evenly text-lg md:text-xl p-6 space-y-6"
   >
-    <template v-if="idGeneration">
+    <div
+      v-if="idGeneration"
+      class="bg-primary-hover py-10 rounded-md shadow-sm"
+    >
       <p>
-        Your ID is <b>{{ userData["user_id"] }}</b>
+        <b>Your ID is {{ userData["user_id"] }}</b>
       </p>
-      <p>Please note this down. Use this to sign-in going forward.</p>
-    </template>
+      <p>
+        Kindly make a note of it. You will need this to log in to all your
```

Comment:
wouldnt this require translations too

Reply:
Haven't gotten it. When I do, I'll add it.